packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -305,20 +308,39 @@ export default function LabeledField(props: Props) {
                     aria-label={labels.readOnlyIconAriaLabel}
                     color={semanticColor.core.foreground.neutral.subtle}
                 />
-                <BodyText
-                    style={[styles.textWordBreak, styles.helperTextMessage]}
-                >
-                    {readOnlyMessage}
-                </BodyText>
+                <BodyText style={styles.helperText}>{readOnlyMessage}</BodyText>
             </View>
         );
     }
 
+    function maybeRenderAdditionalHelperMessage() {
+        if (!additionalHelperMessage) {
+            return null;
+        }
+
+        return (
+            <BodyText
+                id={additionalHelperMessageId}
+                testId={testId && `${testId}-additional-helper-message`}
+                style={[
+                    styles.helperText,
+                    styles.spacingAboveHelperText,
+                    isDisabled && styles.disabledHelperText,
+                    stylesProp?.additionalHelperMessage,
+                ]}
+                tag="div"
+            >
+                {additionalHelperMessage}
```

Comment:
Are you counting error and read-only as any other kinds of helper messages? I'm not sure if we need the `additional` part in the prop name, but you have more context so I'd love to get your thoughts on that.

Reply:
I'll continue with the review then.